packages/components/src/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,287 @@
+                    <FormControl
+                        id="pluralItemName"
+                        type="text"
+                        placeholder="Enter a plural name for this Issue"
+                        value={value}
+                        onChange={onInputChange}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class CommentSortDirectionDropDown extends React.PureComponent<IssuesListDefBasicPropertiesInputsProps> {
+
+    getHelpTip() {
+        return ISSUES_LIST_DEF_SORT_DIRECTION_TIP;
+    }
+
+    onChange = (name: string, formValue: any, selected: String, ref: any): any => {
+        if (selected && this.props.onSelect) {
+            this.props.onSelect(selected, name);
+        }
+    };
+
+    render() {
+        const { model } = this.props;
+
+        let sortDirectionOptions = [];
+        sortDirectionOptions.push({label: "Oldest first", id: "ASC"});
+        sortDirectionOptions.push({label: "Newest first", id: "DESC"});
+
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Comment sort direction" helpTipBody={this.getHelpTip} required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <SelectInput
+                        name={"commentSortDirection"}
+                        options={sortDirectionOptions}
+                        inputClass={'col-xs-12'}
+                        valueKey={'id'}
+                        onChange={this.onChange}
+                        value={model.commentSortDirection ? model.commentSortDirection : sortDirectionOptions[0]}
+                        formsy={false}
+                        showLabel={true}
+                        multiple={false}
+                        required={false}
+                        placeholder={''}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class AssignedToGroupInput extends React.PureComponent<SecurityUserGroupProps, any> {
+
+    getHelpTip() {
+        return ISSUES_LIST_GROUP_ASSIGN_TIP;
+    }
+
+    onChange = (name: string, formValue: any, selected: Principal, ref: any): any => {
+        if (selected && this.props.onSelect) {
+            this.props.onSelect(selected, name);
+        }
+    };
+
+    render() {
+        const { model, coreGroups } = this.props;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Assigned to list comes from" helpTipBody={this.getHelpTip} required={false} />
+                </Col>
+                <Col xs={9} lg={8}>
+                    <SelectInput
+                        name={'assignedToGroup'}
+                        options={coreGroups.toArray()}
+                        placeholder=""
```

Comment:
To match the GWT behavior and the text in the help tip, this placeholder text could be "All Project Users", which would give the user an indicator of what the behavior will be if they don't select anything from this input.

Reply:
done